engine/univMinim.ml

```diff
@@ -267,12 +267,16 @@ let minimize_univ_variables ctx us algs left right cstrs =
 module UPairs = OrderedType.UnorderedPair(Univ.Level)
 module UPairSet = Set.Make (UPairs)
 
+let is_bound l lbound = match lbound with
+| UGraph.Bound.Prop -> Level.is_prop l
+| UGraph.Bound.Set -> Level.is_set l
```

Comment:
Why not have this in ugraph?

Reply:
I prefer not having people rely on this too much, the whole mechanism is fairly hackish.